save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FileUploader.kt

```diff
@@ -224,56 +274,67 @@ private fun fileUploader() = FC<UploaderProps> { props ->
                             disabled = true
                             +"Select a file from existing"
                         }
-                        props.availableFiles.sortedByDescending { it.key.uploadedMillis }.map {
+
+                        availableFiles.sortedByDescending {
+                            if (it is FileInfo) {
+                                it.key.uploadedMillis
+                            } else {
+                                it.sizeBytes
+                            }
+                        }.map {
                             option {
                                 className = ClassName("list-group-item")
-                                value = it.key.name
+                                value = it.name
```

Comment:
it can be issue for FileInfo -- we can have two files with similar name,but different uploadedMillis. will be selected first matched file

Reply:
Now use `fileInfoToPrettyPrint` method from Props for that also